unity-renderer/Assets/DCLPlugins/ECS7/ScenesUI/ScenesUiPanelSettings.asset

```diff
@@ -15,14 +15,14 @@ MonoBehaviour:
   themeUss: {fileID: -4733365628477956816, guid: de12f39179342b6408bb91a479f95503,
     type: 3}
   m_TargetTexture: {fileID: 0}
-  m_ScaleMode: 1
+  m_ScaleMode: 0
```

Comment:
Could you include an assertion to check that this value is always 0? to prevent accidentally changing the .asset yaml